test/dummy.go

```diff
@@ -82,7 +82,10 @@ func (d *DummyDA) Get(ctx context.Context, ids []da.ID, _ da.Namespace) ([]da.Bl
 func (d *DummyDA) GetIDs(ctx context.Context, height uint64, _ da.Namespace) ([]da.ID, error) {
 	d.mu.Lock()
 	defer d.mu.Unlock()
-	kvps := d.data[height]
+	kvps, ok := d.data[height]
+	if !ok {
+		return nil, errors.New("no blobs at given height")
```

Comment:
nitpick: turn this error into sentinel value and use it in tests (I hope this doesn't sound like over-engineering :P)

Reply:
was referencing to other inline errors tbh
will make it sentinel yes